python/nvfuser_next/__init__.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+import sys
+import torch
+
+# This is needed when libnvfuser_next.so is patched and doesn't have the pytorch library location available.
+pytorch_lib_dir = os.path.join(os.path.dirname(torch.__file__), "lib")
+if pytorch_lib_dir not in sys.path:
+    sys.path.append(pytorch_lib_dir)
+
```

Comment:
Naive question. Are we expecting people to import both nvfuser and nvfuser_next?

The way things are imported makes me worried about name conflicts.
i.e. should we assert on not seeing nvfuser in `sys.modules` when importing nvfuser_next?

Reply:
> Are we expecting people to import both nvfuser and nvfuser_next?

No.

> should we assert on not seeing nvfuser in sys.modules when importing nvfuser_next?

Yes.

Reply:
@jjsjann123 I added assertions to `nvfuser/__init__.py` and `nvfuser_next/__init__.py` and python tests to check for the conflicts.